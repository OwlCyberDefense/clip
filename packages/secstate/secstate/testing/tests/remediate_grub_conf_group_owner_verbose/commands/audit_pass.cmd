/usr/sbin/secstate audit grub-conf
