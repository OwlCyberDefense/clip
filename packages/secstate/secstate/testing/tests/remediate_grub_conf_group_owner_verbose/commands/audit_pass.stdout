--Results for 'grub-conf' (Profile: 'None')--
Passed:         1
Mitigated:      0
Failed:         0
Fixed:          0
Not Selected:   0
Not Checked:    0
Not Applicable: 0
Error:          0
Informational:  0
Unknown:        0
Parse Error:    0
Total:          1

Generating HTML output...
